components/dash-core-components/dash_core_components_base/__init__.py

```diff
@@ -121,15 +122,6 @@
             "namespace": "dash",
             "dynamic": True,
         },
-        {
-            "relative_package_path": "dcc/plotly.min.js",
-            "external_url": (
-                "https://unpkg.com/dash-core-components@{}"
-                "/dash_core_components/plotly.min.js"
-            ).format(__version__),
-            "namespace": "dash",
-            "async": "eager",
-        },
```

Comment:
Can we also delete the following two entries, `async-plotlyjs.js` and its `.map`? And I guess we can remove the `prebuild:js` command in dcc's `package.json` that creates the `plotly.min.js` file referenced here, right?